src/nidigital/examples/nidigital_ppmu_source_and_measure.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import sys
+import time
+
+
+def example(resource_name, options, channels, aperture_time, settling_time, current_level_range, current_level,
+            voltage_limit_high, voltage_limit_low):
+
+    with nidigital.Session(resource_name=resource_name, options=options) as session:
+
+        dir = os.path.join(os.path.dirname(__file__), 'ppmu_source_and_measure_files')
+
+        # Load pin map (.pinmap) created using Digital Pattern Editor
+        pin_map_filename = os.path.join(dir, 'PinMap.pinmap')
+        session.load_pin_map(pin_map_filename)
+
+        # Configure PPMU measurements
+        session.channels[channels].ppmu_aperture_time = aperture_time
+        session.channels[channels].ppmu_aperture_time_units = nidigital.PPMUApertureTimeUnits.SECONDS
+
+        session.channels[channels].ppmu_output_function = nidigital.PPMUOutputFunction.CURRENT
```

Comment:
I think we want to demonstrate sourcing both DC voltage and DC current. The configuration for both involves a set of properties and they are subtly different between each other.

(The args for the two cases can be handled via 2 sub-commands).